elasticsearch-index-sink.kamelet.yaml

```diff
@@ -0,0 +1,76 @@
+spec:
+  sources:
+    - content: |-
+        public class ESWorkaround extends org.apache.camel.builder.RouteBuilder {
+          @Override
+          public void configure() throws Exception {
+            getContext().setAutowiredEnabled(false);
+          }
+        }
+      name: ESWorkaround.java
+  definition:
+    title: "ElasticSearch Index Sink"
+    description: |-
+      Insert data into ElasticSearch. Input data must have JSON format.
+    required:
+      - query
+      - clusterName
+      - indexName
+      - hostAddresses
+    properties:
+      user:
+        title: Username
+        description: Username to connect to ElasticSearch.
+        type: string
+      password:
+        title: Password
+        description: Password to connect to ElasticSearch.
+        type: string
+      enableSSL:
+        title: Enable SSL
+        description: Do we want to connect using SSL?
+        type: boolean
+        default: true
+      hostAddresses:
+        title: Host addresses
+        description: Comma separated list with ip:port formatted remote transport addresses to use.
+        type: string
+      indexName:
+        title: Index in ElasticSearch
+        description: The name of the index to act against.
+        type: string
+      clusterName:
+        title: ElasticSearch cluster name
+        description: Name of the cluster.
+        type: string
+  dependencies:
+    - "camel:elasticsearch-rest"
+    - "camel:gson"
+  types:
+    out:
```

Comment:
Can you declare the in type as JSON as well?

Reply:
Done.